### PR TITLE
add `store_picture` variable

### PR DESCRIPTION
**Pull request by p0lycarpio**
Added the `store_picture` variable in the `item` model to be able to display the store logo in the notifications or webhooks.

### CODE REVIEW

**Review by Der-Henning — COMMENTED** [review]
Hi @p0lycarpio.
Thank you for your contribution.
I think we should add `store_cover` and `store_logo` as options.
And what about `item_cover` and `item_logo`? They seem to be the same as the store pictures. Or do you have an example where they differ?

Is `"-"` a reasonable default for an url?

**Comment by p0lycarpio** [comment]
Hello @Der-Henning,
Indeed the `logo` and `cover` images look the same in `item` and `store`. I will take those of the item.

As for the default, the API seems to always return an image. For a store that does not have a logo, an image is generated with its initials. For the cover, there are several default values like : https://images.tgtg.ninja/standard_images/GENERAL/other1.jpg
What URL can I set for both images?

File: src/models/item.py [hunk 1]
```diff
@@ -41,6 +41,10 @@ def __init__(self, data: dict):
                       10**price_including_taxes.get("decimals", 0))
         self.price = f"{self.price:.2f}"
         self.currency = item.get("price_including_taxes", {}).get("code", "-")
+        self.item_logo = item.get("logo_picture", {}).get(
+            "current_url", "https://tgtg-mkt-cms-prod.s3.eu-west-1.amazonaws.com/13512/TGTG_Icon_White_Cirle_1988x1988px_RGB.png")
+        self.item_cover = item.get("cover_picture", {}).get(
+            "current_url", "https://images.tgtg.ninja/standard_images/GENERAL/other1.jpg")
```
Comment (Der-Henning) [inline src/models/item.py:47]:
```suggestion
        self.item_logo = item.get("logo_picture", {}).get(
            "current_url",
            "https://tgtg-mkt-cms-prod.s3.eu-west-1.amazonaws.com/"
            "13512/TGTG_Icon_White_Cirle_1988x1988px_RGB.png")
        self.item_cover = item.get("cover_picture", {}).get(
            "current_url",
            "https://images.tgtg.ninja/standard_images/GENERAL/other1.jpg")
```
Fixing E501 line too long (> 79 characters)

**Review by Der-Henning — CHANGES_REQUESTED** [review]
This looks fine. Please accept the changes to comply with the PEP8 style guide. Thank you.

**Review by Der-Henning — APPROVED** [review]

**Pull request merged**